pkg/detectors/gitlab/gitlabv2_test.go

```diff
@@ -1,115 +0,0 @@
-package gitlab
-
-import (
-	"context"
-	"fmt"
-	"testing"
-	"time"
-
-	"github.com/kylelemons/godebug/pretty"
-	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
-	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
-	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
-)
-
-func TestGitlabv2_FromChunk(t *testing.T) {
```

Comment:
Why did you delete this test?

Reply:
because it was testing v2. i figured we should just have a test for v1 and a test for v2, not both. but i'm open to thoughts on if we should keep it

Reply:
oh, i totally missed that it was a v2 test in the v1 directory. now that i see that, i completely agree.